gettsim/policy_for_date.py

```diff
@@ -161,3 +161,32 @@ def load_ges_renten_vers_params(raw_pension_data, actual_date):
                     policy_year
                 ]["value"]
     return pension_data
+
+
+def load_regrouped_wohngeld(year, month=1, day=1):
+    tax_data_raw = yaml.safe_load((ROOT_DIR / "data" / "wohngeld_neu.yaml").read_text())
+
+    actual_date = datetime.date(year=year, month=month, day=day)
+    tax_data = {}
+    for param in tax_data_raw:
+        policy_dates = sorted(
+            key for key in tax_data_raw[param].keys() if type(key) == datetime.date
+        )
+        past_policies = [x for x in policy_dates if x <= actual_date]
+        if not past_policies:
+            # TODO: Should there be missing values or should the key not exist?
+            tax_data[param] = np.nan
+        else:
+            policy_in_place = tax_data_raw[param][np.max(past_policies)]
+            if "scalar" in policy_in_place.keys():
+                tax_data[param] = policy_in_place["scalar"]
+            else:
+                value_keys = sorted(
+                    key for key in policy_in_place.keys() if type(key) == int
+                )
+                tax_data[param] = {}
```

Comment:
So thats the reason why I used value. I don't know a better way. Also googled quite a bit. Maybe you could point me in a better direction. @hmgaudecker

Reply:
Not sure i understand where the problem is... We do need some specialised code here, hence the `type` attribute (holding `piecewise_linear` and friends). I have not looked at the Wohngeld formula, not sure how to best call that or whether it makes any sense to give it a generic name.